pytest/tests/sanity/sync_ban.py

```diff
@@ -0,0 +1,89 @@
+import sys, time, functools, asyncio
```

Comment:
Describe what the test does in a comment at the beginning. It's very hard later (or even now) to read tests without understanding what is the intended behavior and what exactly they test.